File: uri/1 - Basics/1012/Main.rb
def main()
    
    pi = 3.14159
    ar=gets.split()
    a=ar[0].to_f
    b=ar[1].to_f
    c=ar[2].to_f

    print("TRIANGULO: %.3f\n" % ((a*c)/2.0) )
    print("CIRCULO: %.3f\n" % (pi*c*c) )
    print("TRAPEZIO: %.3f\n" % (((a+b)*c)/2.0) )
    print("QUADRADO: %.3f\n" % (b*b) )
    print("RETANGULO: %.3f\n" % (a*b) )
end

main()

File: uri/1 - Basics/1012/Main.cs
using System;

class URI
{

	static void Main(string[] args)
	{

		string[] vet = Console.ReadLine().Split(' ');
		float a = float.Parse(vet[0]);
		float b = float.Parse(vet[1]);
		float c = float.Parse(vet[2]);

		const double pi = 3.14159;

		Console.WriteLine($"TRIANGULO: {((a * c) / 2.0):F3}");
		Console.WriteLine($"CIRCULO: {((pi * c * c)):F3}");
		Console.WriteLine($"TRAPEZIO: {(((a + b) * c) / 2.0):F3}");
		Console.WriteLine($"QUADRADO: {(b * b):F3}");
		Console.WriteLine($"RETANGULO: {(a * b):F3}");
	}

}
